charts/nri-kube-events/templates/NOTES.txt

```diff
@@ -1,16 +1 @@
-{{- if (include "nri-kube-events.areValuesValid" .) }}
-Your deployment of New Relic Kube Events is complete.
-{{- else -}}
-##############################################################################
-####       ERROR: You did not set a licenseKey and/or cluster name.       ####
-##############################################################################
 
```

Comment:
any reason to keep the file empty?

Reply:
Nope, I thought it was "kind-of-our-standard" and even if empty, it was a good practice leave it. Going to updated.

Reply:
Ditto.